superdesk/metadata/item.py

```diff
@@ -298,6 +298,11 @@
     'associations': {
         'type': 'dict',
     },
+    'alt_text': {
+        'type': 'string',
+        'nullable': True,
+        'mapping': not_analyzed,
```

Comment:
so we use it for eg. ids, because when analyzed it will split `localhost:2012-12:123` into `localhost` `2012` `12` and `123`, so then that id would match search for localhost, 2012, etc.
if it's some text which is not a single value, it should be analyzed in general. unless you want to ignore it (not to be able to search by that), but there is different setting for that